src/components/Collections/CollectionOptions/QuickOptions/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { CollectionActions } from '..';
+import React from 'react';
+import { CollectionSummaryType } from 'constants/collection';
+import { FlexWrapper } from 'components/Container';
+import constants from 'utils/strings/constants';
+import { MoveToTrashOption } from './MoveToTrashOption';
+import { DownloadOption } from './DownloadOption';
+import { ShareOption } from './ShareOption';
+import {
+    showDownloadQuickOption,
+    showShareQuickOption,
+    showTrashQuickOption,
+} from 'utils/collection';
+interface Iprops {
+    handleCollectionAction: (
+        action: CollectionActions,
+        loader?: boolean
+    ) => (...args: any[]) => Promise<void>;
+    collectionSummaryType: CollectionSummaryType;
+}
+
+export function QuickOptions({
+    handleCollectionAction,
+    collectionSummaryType,
+}: Iprops) {
+    return (
+        <FlexWrapper sx={{ gap: '16px' }}>
+            {showTrashQuickOption(collectionSummaryType) && (
+                <MoveToTrashOption
+                    handleCollectionAction={handleCollectionAction}
+                />
+            )}
+            {showDownloadQuickOption(collectionSummaryType) && (
+                <DownloadOption
+                    handleCollectionAction={handleCollectionAction}
+                    tooltipTitle={
+                        collectionSummaryType ===
+                        CollectionSummaryType.favorites
```

Comment:
can we move this logic to inside the `DownloadOption` component? we can pass it to the `collectionSummaryType`. can it can do this logic and show the appropriate tooltip